libraries/provision/ansible/playbooks/install-couchbase-server-package.yml

```diff
@@ -75,9 +75,13 @@
       shell: "{{ couchbase_server_home_path }}/bin/couchbase-cli cluster-init -c {{ couchbase_server_node }}:{{ couchbase_server_admin_port }} --user={{ couchbase_server_admin }} --password={{ couchbase_server_password }} --cluster-port={{couchbase_server_admin_port}} --cluster-ramsize={{ couchbase_server_cluster_ram }}  --services=data,index,query  --cluster-index-ramsize={{ couchbase_server_index_ram }}"
       when:  "'4.0' in couchbase_server_package_name or '4.1' in couchbase_server_package_name"
 
-    - name: COUCHBASE SERVER | Configure cluster settings (4.5.X and up)
+    - name: COUCHBASE SERVER | Configure cluster settings (4.5.X and 4.6.X)
       shell: "{{ couchbase_server_home_path }}/bin/couchbase-cli cluster-init -c {{ couchbase_server_node }}:{{ couchbase_server_admin_port }} --user={{ couchbase_server_admin }} --password={{ couchbase_server_password }} --cluster-port={{couchbase_server_admin_port}} --cluster-ramsize={{ couchbase_server_cluster_ram }}  --services=data,index,query  --cluster-index-ramsize={{ couchbase_server_index_ram }}  --index-storage-setting=default"
-      when:  "not '4.0' in couchbase_server_package_name and not '4.1' in couchbase_server_package_name"
+      when:  "'4.5' in couchbase_server_package_name or '4.6' in couchbase_server_package_name"
+
+    - name: COUCHBASE SERVER | Configure cluster settings (4.7.X)
+      shell: "{{ couchbase_server_home_path }}/bin/couchbase-cli cluster-init -c {{ couchbase_server_node }}:{{ couchbase_server_admin_port }} --cluster-username={{ couchbase_server_admin }} --cluster-password={{ couchbase_server_password }} --cluster-port={{couchbase_server_admin_port}} --cluster-ramsize={{ couchbase_server_cluster_ram }}  --services=data,index,query  --cluster-index-ramsize={{ couchbase_server_index_ram }}  --index-storage-setting=default"
+      when:  "'4.7' in couchbase_server_package_name"
```

Comment:
Can we make this default? For example, a future 4.8 and 4.9 release would break under this current logic? It would be something like "not 4.5" and "not 4.6" ...